mailpoet/lib/AdminPages/Pages/Automation.php

```diff
@@ -43,6 +50,12 @@ public function render() {
         'nonce' => $this->wp->wpCreateNonce('wp_rest'),
       ],
       'workflowCount' => $this->workflowStorage->getWorkflowCount(),
+      'templates' => array_map(
+        function(WorkflowTemplate $template): array {
+          return $template->toArray();
```

Comment:
Ideally, we should create a template mapper as `toArray` is mainly for the DB. But good for now.